README

```diff
@@ -130,6 +130,11 @@ METHODS
 
     return the current out queue length.
 
+  flush
+        $kafka->flush($timeout_ms)
+
+    wait until all outstanding produce requests, et.al, are completed.
+
```

Comment:
For documentation, please update the POD in `lib/Kafka/Librd.pm` instead of the `README`.  The latter is generated from the former as part of the release process.